src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -72,7 +77,8 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
         String author = UserStore.getInstance()
           .getUser(message.getAuthorId()).getName();
     %>
-      <li><strong><%= author %>:</strong> <%= message.getContent() %></li>
+    <%--  <li><strong><%= author %>:</strong> <%= message.getContent() %></li> --%>
+      <li><strong><a href="/profile/<%=request.getSession().getAttribute("author")%>"><%= author %>:</a></strong> <%= message.getContent() %></li>
```

Comment:
I think you can use the `author` variable in the link here, instead of getting an attribute from the session.